test/parallel/test-fs-promises-writefile.js

```diff
@@ -55,4 +111,9 @@ doWrite()
   .then(doAppend)
```

Comment:
This whole then chain could be refactored into a utility async function using `await`s... would likely be nicer.

Reply:
@jasnell 
Thanks, I fixed it.